pycparser/c_parser.py

```diff
@@ -920,9 +940,20 @@ def p_specifier_qualifier_list_3(self, p):
     def p_specifier_qualifier_list_4(self, p):
         """ specifier_qualifier_list  : type_qualifier_list type_specifier
         """
-        spec = dict(qual=p[1], storage=[], type=[], function=[])
+        spec = dict(qual=p[1], alignment=[], storage=[], type=[], function=[])
```

Comment:
So you really need _add_declaration_specifier here? Can't you just copy the `type`?

Reply:
Rebased on top of master fixing. Indeed no need.